data/opensource/src/androidMain/kotlin/app/tivi/data/opensource/store/AndroidOpenSourceStore.kt

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023, Google LLC, Christopher Banes and the Tivi project contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package app.tivi.data.opensource.store
+
+import android.app.Application
+import app.tivi.data.opensource.OpenSourceItem
+import app.tivi.data.opensource.OpenSourceState
+import app.tivi.data.opensource.SimpleOpenSourceState
+import java.io.IOException
+import java.io.InputStream
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.emptyFlow
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import me.tatarka.inject.annotations.Inject
+
+@Inject
+class AndroidOpenSourceStore(private val context: Application) : OpenSourceStore {
```

Comment:
Only the `getOpenSourceItemList()` function here will actually be different on each platform. Please move this to `commonMain` and then extract a `LicensesFetcher` interface which is responsible for loading the JSON file and returning a `List<OpenSourceItem>`. Each platform will need it's own `LicensesFetcher` impl.